beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -187,8 +184,7 @@ func (s *InitialSync) roundRobinSync(genesis time.Time) error {
 	// mitigation. We are already convinced that we are on the correct finalized chain. Any blocks
 	// we receive there after must build on the finalized chain or be considered invalid during
 	// fork choice resolution / block processing.
-	best := bestPeer()
```

Comment:
Why change this?

Reply:
i had a case where the chosen peer had failure on these slots. then it all fails

Reply:
better not to be dependent on a single node